app/controllers/cleaners_controller.rb

```diff
@@ -25,9 +28,12 @@ def edit
   # POST /cleaners.json
   def create
     @cleaner = Cleaner.new(cleaner_params)
-
+    @cleaner_cities = params[:city_ids]
     respond_to do |format|
       if @cleaner.save
+        @cleaner_cities.each do |city|
```

Comment:
@krupa147 This could be a new private method - Single responsibility and separation of concern.